xbmc/utils/StringUtils.cpp

```diff
@@ -336,76 +176,100 @@ int compareWchar (const void* a, const void* b)
   return 0;
 }
 
-wchar_t tolowerUnicode(const wchar_t& c)
+std::string transformToLower(std::string_view input, std::locale loc)
 {
-  wchar_t* p = (wchar_t*) bsearch (&c, unicode_uppers, sizeof(unicode_uppers) / sizeof(wchar_t), sizeof(wchar_t), compareWchar);
-  if (p)
-    return *(unicode_lowers + (p - unicode_uppers));
-
-  return c;
+  const std::ctype<char>& ct = use_facet<std::ctype<char> >(loc);
```

Comment:
`use_facet` should be in the `std` namespace, `std::use_facet`

ref: https://en.cppreference.com/w/cpp/locale/use_facet

Reply:
Apparently I don't have my build config set to consider this as an error. Good catch. I have reworked this to not to be so fancy (I have not mastered C++ to be good at facets, etc.. Especially when I would like pointers to instances of functions which use a template to specify both locale and tolower, etc..)